### PR TITLE
mgr/ssh: implement 'service ls'

### PR DESCRIPTION
**Pull request by liewegas**
This is a kind of lame synchronous implementation because I don't really understand how to use the completions.  In particular, various other mgr/ssh  methods (update_*) will want to get this same inventory (and block while we getch it), but the orchestrator_wait method is implmented in orchestrator_cli... should we just copy it here?

Anyway, this is start and it works!

```
$ bin/ceph orchestrator service ls 
  type    id     host    container     version    status  description 
 crash  glitch  glitch   <unknown>    <unknown>   error      error    
  mgr   glitch  glitch  5ca676762a01    14.2.4   running    running   
```

### CODE REVIEW

File: src/pybind/mgr/ssh/module.py [hunk 1]
```diff
@@ -345,7 +377,55 @@ def get_hosts(self):
...
+                    sd.service_instance = host  # e.g., crash
+                if service_id and service_id != sd.service_instance:
+                    continue
+                sd.nodename = host
+                sd.container_id = d['container_id']
+                sd.version = d['version']
+                if d['active']:
+                    sd.status = 1
+                else:
+                    sd.status = 0   # FIXME what about failed?
```
Comment (sebastian-philipp) [inline src/pybind/mgr/ssh/module.py]: ` FIXME what about failed?` yep, we need some kind of error message in `ServiceDescription`

Comment (liewegas) [reply src/pybind/mgr/ssh/module.py]: my TODO here is to replace the 'active' column with a 'status' enum: running, stopped, failed or similar.  I need to look more closely at how the systemd states compare to k8s.

Comment (liewegas) [reply src/pybind/mgr/ssh/module.py]: FWIW i'm not really sure what service_description should have in general...

Comment (liewegas) [reply src/pybind/mgr/ssh/module.py]: fixed

**Pull request merged**